packages/react-native-web/src/modules/usePlatformMethods/index.js

```diff
@@ -43,22 +44,30 @@ function setNativeProps(node, nativeProps, classList, pointerEvents, style, prev
  * Adds non-standard methods to the hode element. This is temporarily until an
  * API like `ReactNative.measure(hostRef, callback)` is added to React Native.
  */
-export default function usePlatformMethods(props: Object) {
+export default function usePlatformMethods({
+  classList,
+  pointerEvents,
+  style
+}: {
+  classList?: Array<string | boolean>,
+  style?: $PropertyType<ViewProps, 'style'>,
+  pointerEvents: $PropertyType<ViewProps, 'pointerEvents'>
+}) {
   const previousStyleRef = useRef(null);
-  const { classList, style, pointerEvents } = props;
+  const setNativePropsArgsRef = useRef(null);
+  setNativePropsArgsRef.current = { classList, pointerEvents, style };
 
-  return useMemo(
-    () => (hostNode: any) => {
-      if (hostNode != null) {
-        hostNode.measure = callback => UIManager.measure(hostNode, callback);
-        hostNode.measureLayout = (relativeToNode, success, failure) =>
-          UIManager.measureLayout(hostNode, relativeToNode, failure, success);
-        hostNode.measureInWindow = callback => UIManager.measureInWindow(hostNode, callback);
-        hostNode.setNativeProps = nativeProps =>
-          setNativeProps(hostNode, nativeProps, classList, pointerEvents, style, previousStyleRef);
-      }
-      return hostNode;
-    },
-    [classList, pointerEvents, style]
-  );
+  return useRef((hostNode: any) => {
+    if (hostNode != null) {
+      hostNode.measure = callback => UIManager.measure(hostNode, callback);
+      hostNode.measureLayout = (relativeToNode, success, failure) =>
+        UIManager.measureLayout(hostNode, relativeToNode, failure, success);
+      hostNode.measureInWindow = callback => UIManager.measureInWindow(hostNode, callback);
+      hostNode.setNativeProps = nativeProps => {
+        const { classList, style, pointerEvents } = setNativePropsArgsRef.current || {};
+        setNativeProps(hostNode, nativeProps, classList, pointerEvents, style, previousStyleRef);
+      };
+    }
+    return hostNode;
+  }).current;
```

Comment:
This no longer returns a ref. I don't think we want to set the initial value of the ref to be a function either. The code that was deleted was memoizing a function ref.